pydropsonde/circles.py

```diff
@@ -16,5 +21,138 @@ class Circle:
     segment_id: str
 
     def dummy_circle_function(self):
```

Comment:
once there are real functions, we can remove this. it was just to test that the iteration worked